core/test/Utils.sol

```diff
@@ -14,3 +14,10 @@ function deploySingleLender(ERC20 asset, address treasury, RateModel rateModel)
     Lender(proxy).initialize(rateModel, 8);
     return Lender(proxy);
 }
+
+contract Router {
```

Comment:
It's because we're using a push architecture; i.e. by the time `deposit` or `repay` is called, `Lender` expects to have already been paid (it doesn't attempt to pull tokens in from the sender via `erc20.transferFrom`). Therefore, we need a router to help send tokens to the `Lender` right before each deposit.

Reply:
Technically we're supposed to support both flows; see #33